monkey/monkey_island/cc/ui/src/components/configuration-components/ValidationFormats.js

```diff
@@ -1,13 +1,22 @@
 const ipRegex = '((25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\\.){3}(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)'
 const cidrNotationRegex = '([0-9]|1[0-9]|2[0-9]|3[0-2])'
 const hostnameRegex = '^([A-Za-z0-9]*[A-Za-z]+[A-Za-z0-9]*.?)*([A-Za-z0-9]*[A-Za-z]+[A-Za-z0-9]*)$'
+// path starts with `/` OR `$`
+const linuxDirRegex = '^/|\\$'
```

Comment:
`afdbfasd$` is a valid absolute path according to this regex. Maybe change to `^/|^\\$`?